app/utils/whitelabel.ts

```diff
@@ -1,31 +1,121 @@
+import AskDarcelImage from '../assets/img/askdarcel.svg';
+import BackgroundImage from '../assets/img/bg.png';
+import SearchByAlgoliaImage from '../assets/img/search-by-algolia.png';
+import SFFamiliesLogo from '../assets/img/sf-families.svg';
+import SFServiceLogo from '../assets/img/sf-service.svg';
+import SFSeal from '../assets/img/sf-seal.png';
 import config from '../config';
+import styles from '../components/ui/Navigation/Navigation.module.scss';
 
-/** Whether we should display the SF Service Guide branded version of the site */
-const isSFFamiliesSite = () => window.location.host.indexOf(config.SFFAMILIES_DOMAIN) > -1;
-
-const isSFServiceGuideSite = () => !isSFFamiliesSite()
-  && window.location.host.indexOf(config.MOHCD_DOMAIN) > -1;
-
-const getSiteTitle = () => {
-  if (isSFServiceGuideSite()) {
-    return 'SF Service Guide';
-  }
-  if (isSFFamiliesSite()) {
-    return 'SF Families';
-  }
-  return 'AskDarcel';
-};
-
-const getSiteUrl = () => {
-  if (isSFServiceGuideSite()) {
-    return 'https://sfserviceguide.org';
-  }
-  if (isSFFamiliesSite()) {
-    return 'https://sffamilies.sfserviceguide.org/';
-  }
-  return 'https://askdarcel.org';
-};
-
-export {
-  isSFServiceGuideSite, isSFFamiliesSite, getSiteTitle, getSiteUrl,
-};
+// Include new white label here
+type WhiteLabelSiteKey = 'defaultWhiteLabel' | 'SFServiceGuide' | 'SFFamilies';
+
+interface WhiteLabelSite {
+  appImages: {
+    background: string;
+    logoLarge: string;
+    logoSmall: string;
+    algolia: string;
+    mohcdSeal: string;
+  };
+  intercom: boolean;
+  logoLinkDestination: string;
+  navLogoStyle: string;
+  showBanner: boolean;
+  showMobileNav: boolean;
+  showSearch: boolean;
+  siteNavStyle: string;
+  siteUrl: string;
+  title: string;
+  userWay: boolean;
+}
+
+// Include a domain in config.js
+function determineWhiteLabelSite(): WhiteLabelSiteKey {
+  if (window.location.host === config.SFFAMILIES_DOMAIN) return 'SFFamilies';
+  if (window.location.host === config.MOHCD_DOMAIN) return 'SFServiceGuide';
+  return 'defaultWhiteLabel';
+}
+
+const configKey = determineWhiteLabelSite();
+
+/*
+  Specify what is viewed in each white label.
+  A '/' (which is a forward-slash) as a value for logoLinkDestination
+  denotes that the link is internal to the application.
```

Comment:
This comment would be better to place on the interface definition, right above the actual `logoLinkDestination`, since that interface should be the one place where we define what the interface is.